app/build.gradle

```
@@ -40,9 +40,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.2.0'
     implementation 'com.google.android.material:material:1.1.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.2'
-    implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
+    implementation 'androidx.recyclerview:recyclerview:1.0.0'
```

```suggestion
    implementation 'androidx.recyclerview:recyclerview:1.1.0'
```
Let's not add obsolete dependencies.